docs/plans/2026-05-02-feat-archscope-product-sharpness-and-agent-ux-plan.md

```
@@ -0,0 +1,678 @@
+---
+schema_version: 1
+title: "feat: Archscope product sharpness and agent UX plan"
+type: feat
+status: draft
+date: 2026-05-02
+origin: docs/specs/2026-05-02-feat-archscope-product-sharpness-and-agent-ux-spec.md
+spec: docs/specs/2026-05-02-feat-archscope-product-sharpness-and-agent-ux-spec.md
+source_spec: docs/specs/2026-05-02-feat-archscope-product-sharpness-and-agent-ux-spec.md
+plan_route: fresh
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)